invoicing_app/tests/tests.py

```diff
@@ -235,27 +254,44 @@ def test_generate_tax_receipts_w_1_po_CL(self, patch_call):
         )
         my_pay_opt_2.save()
 
-        # Set the mg_fsee in 11.0 to see that the element that goes to the service isn't this
+        # Set the mg_fee in 11.0 to see that the element that goes to the service isn't this
         my_order_2 = OrderFactory.build(
             event=my_event_2,
             mg_fee=11.0
         )
         my_order_2.save()
 
         self.my_command.handle(**self.options)
+        self.my_command_new.handle(**self.options)
+
+        # Expected is 400 because mg_fee is 5.1 and eb_tax = 1.1
+        expected_tot_tax_amount = 400
+        call_once = 1
+
         self.assertEqual(
             patch_call.call_count,
-            1
+            call_once
```

Comment:
I believe you could use assert_called_once. See https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once